docs/source/purldb/rest_api.rst

```diff
@@ -834,6 +866,87 @@ https://scancodeio.readthedocs.io/en/latest/built-in-pipelines.html.
             "history": "https://public.purldb.io/api/packages/4f3a57de-e367-43c6-a7f1-51633d0ecd45/history/"
         }
 
+collect actions
+---------------
+
+index_packages
+^^^^^^^^^^^^^^
+
+Take a list of ``packages`` (where each item is a dictionary containing either PURL
+or versionless PURL along with vers range, optionally with source package PURL)
+and index it.
+Also each package can have list of ``addon_pipelines`` to run on the package.
+Find all addon pipelines [here.](https://scancodeio.readthedocs.io/en/latest/built-in-pipelines.html)
+
+
+If ``reindex`` flag is True then existing package will be rescanned, if ``reindex_set``
+is True then all the package in the same set will be rescanned.
+If reindex flag is set to true then all the non existing package will be indexed.
+
+**Note:** When a versionless PURL is supplied without a vers range, then all the versions
```

Comment:
You can have this in a ``note`` directive like this:
https://github.com/nexB/scancode-toolkit/blob/develop/docs/source/getting-started/install.rst?plain=1#L135